src/os/bluestore/ZonedFreelistManager.cc

```diff
@@ -272,6 +272,13 @@ std::vector<zone_state_t> ZonedFreelistManager::get_zone_states(
   return zone_states;
 }
 
+void ZonedFreelistManager::zoned_mark_zone_clean(
```

Comment:
There is pretty perspective WIP to get rid of free list manager updates on each op (alloc or release) and recover allocator state from onodes metadata when needed, see https://github.com/ceph/ceph/pull/39871

The idea behind it to recover allocation information either from onodes metadata (by scanning all of them) after non-graceful shutdown or from a single-shot dump made during a graceful one

Hence wondering if we would be able to modify zoned allocations and avoid DB update on each allocator's op as well?

Reply:
I'm surprised that the IOPS increase is only 25%. I think we can do that once the WIP sorts out all the issues around it and lands.

Reply:
OK, sounds good